src/Components/NotesPage/NotesPage.jsx

```diff
@@ -1,38 +1,85 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useArchive, useAuth, useNotes } from '../../context/';
 import { formatDate } from '../../backend/utils/authUtils';
+import { EditorState, convertToRaw, ContentState } from 'draft-js';
+import 'react-draft-wysiwyg/dist/react-draft-wysiwyg.css';
+import './MyEditor.css';
+import draftToHtml from 'draftjs-to-html';
+import htmlToDraft from 'html-to-draftjs';
+import { MyEditor } from './MyEditor';
+import 'draft-js/dist/Draft.css';
+import './MyEditor.css';
 
 const NotesPage = () => {
-	const { user, encodedToken } = useAuth();
 	const { notes, postNotes, getNotes } = useNotes();
 	const { sendNotesToArchive } = useArchive();
 	const [textField, setTextField] = useState({ title: '', body: '' });
+	const [editorState, setEditorState] = useState(EditorState.createEmpty());
 	const submitNote = () => {
 		postNotes({
 			title: textField.title,
 			body: textField.body,
 			tags: [],
 			createdAt: formatDate(),
 		});
+		setEditorState(EditorState.createEmpty());
+		setTextField({ title: '', body: '' });
 	};
 	const onChangerHandler = (e) => {
```

Comment:
Please rename this to titleChangeHandler.

Reply:
done